pkg/builtin/unary/cot.go

```diff
@@ -0,0 +1,221 @@
+package unary
```

Comment:
could you please add the license header...

Reply:
ok